vmdb/app/models/ems_kubernetes.rb

```diff
@@ -3,6 +3,7 @@ class EmsKubernetes < EmsContainer
   has_many :container_groups,                     :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_replication_controllers,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_namespaces,                 :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
Is a project just an extension of namespace or does it replace namespace?  Do both exist in openshift at the same time?

If it's an extension then I could possibly see using STI, and having one table container_namespaces where the `ContainerProject < ContainerNamespace`.  And then extra fields are just extra and ignored by Namespace usage.

However, generally, when we use STI for modeling providers, we try to keep the base shared and more of an abstract class, and then different providers implement their individual differences as subclasses.  Thus, this possibly should really be:

- ContainerNamespace
  - ContainerNamespaceKubernetes
  - ContainerNamespaceOpenshift

or

- ContainerNamespace
  - ContainerNamespaceKubernetes
  - ContainerProject
    - ContainerProjectOpenshift